app/javascript/packages/document-capture/components/document-capture-warning.tsx

```diff
@@ -118,7 +118,7 @@ function DocumentCaptureWarning({
         troubleshootingOptions={
           <DocumentCaptureTroubleshootingOptions
             location="post_submission_warning"
-            showAlternativeProofingOptions={!isFailedResult}
+            showAlternativeProofingOptions
```

Comment:
It looks like `showAlternativeProofingOptions` is true in all cases now. Should we get rid of the prop altogether?

Reply:
Yeah. Checking out where else it is used, we don't need it anymore.  I will remove it